### PR TITLE
Update NuGet-FAQ.md

### PR DESCRIPTION
**Pull request by diverdan92**
Updating FAQ for ID prefix reservation

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/Policies/NuGet-FAQ.md](https://github.com/diverdan92/docs.microsoft.com-nuget/blob/patch-3/docs/Policies/NuGet-FAQ.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C11%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201710112147454768-504%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=0bt%2Fksj2GLNs8VHH2Im6FXfRKt3v%2FcEtio2iA1dVS%2F4%3D&st=2017-10-11T21%3A43%3A46Z&se=2017-11-11T21%3A48%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/Policies/NuGet-FAQ.md [hunk 1]
```diff
@@ -168,7 +168,9 @@ This is not an issue when using PackageReference, as each project file contains
 
 **Is it possible to reserve names for packages that will be published in future?**
 
-No. If you feel that an existing package has taken the name which suits your package better, try contacting the owner of the package (select on the owner name on the package page on nuget.org). If you don't get a response within a reasonable time, [contact support](https://www.nuget.org/policies/Contact) and we'll look into it.
+You can reserve package ID prefixes for an owner or set of owners on NuGet.org. This will restrict package uploads matching the ID prefix pattern to only be accepted by the owner(s) with the reserved ID. For more information, read the [documentation](https://docs.microsoft.com/nuget/reference/id-prefix-reservation).
+
+If you feel that a previously submitted package has taken the name which suits your package better, try contacting the owner of the package (select on the owner name on the package page on nuget.org). If you don't get a response within a reasonable time, [contact support](https://www.nuget.org/policies/Contact) and we'll look into it.
```
Comment (karann-msft) [inline docs/Policies/NuGet-FAQ.md]: "select on the owner name" .... did you mean to say "select contact owner on the package page" or something simila

Comment (diverdan92) [reply docs/Policies/NuGet-FAQ.md]: I didn't change that part. Do you know how it should actually read?

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/Policies/NuGet-FAQ.md](https://github.com/diverdan92/docs.microsoft.com-nuget/blob/patch-3/docs/Policies/NuGet-FAQ.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C12%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201710121959063819-504%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=yjZ2XAoLfxMNwvAVF4ydskC7lVEHOB0JRNeJhgDrqgg%3D&st=2017-10-12T19%3A55%3A15Z&se=2017-11-12T20%3A00%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**